docs: Agent as Judge (#374)
agno-agi/docs: basics/evals/agent-as-judge/overview.mdx

---
title: Agent as Judge Evals
description: Agent as Judge evals measure custom quality criteria for your Agents and Teams using LLM-as-a-judge methodology.
sidebarTitle: Overview
keywords: [agent as judge evals, custom evaluation, llm as judge, quality evaluation]
---

Agent as Judge evaluations let you define custom quality criteria and use an LLM to score your Agent's responses. You provide evaluation criteria (like "professional tone", "factual accuracy", or "user-friendliness"), and an evaluator model assesses how well the Agent's output meets those standards.

## Basic Example

> In this example, the `AgentAsJudgeEval` will run the Agent with the input, then use an evaluator model to score the Agent's response according to the custom criteria provided.

Reviewer: ```suggestion
In this example, the `AgentAsJudgeEval` will evaluate the output  of the Agent with their input, providing a score of the Agent's response according to the custom criteria provided.
```

```python agent_as_judge.py
from agno.agent import Agent
from agno.db.sqlite import SqliteDb
from agno.eval.agent_as_judge import AgentAsJudgeEval
from agno.models.openai import OpenAIChat

# Setup database to persist eval results
db = SqliteDb(db_file="tmp/agent_as_judge_basic.db")

agent = Agent(
    model=OpenAIChat(id="gpt-4o"),
    instructions="You are a technical writer. Explain concepts clearly and concisely.",
    db=db,
)

response = agent.run("Explain what an API is")

evaluation = AgentAsJudgeEval(
    name="Explanation Quality",
    criteria="Explanation should be clear, beginner-friendly, and use simple language",
    scoring_strategy="numeric",  # Score 1-10
    threshold=7,  # Pass if score >= 7
    db=db,
)

result = evaluation.run(
    input="Explain what an API is",
    output=str(response.content),
    print_results=True,
)

> assert result is not None, "Evaluation should return a result"

Reviewer: You can omit this

Author: Yeah, removed them
